_assets/sass/libraries-main.scss

```diff
@@ -11,4 +11,4 @@
 // app specific styles
 // ====================
 //@import 'apps/ebooks';
-@import 'apps/quicksubmit';
+//@import 'apps/quicksubmit';
```

Comment:
Is there a reason to leave these commented out things in here at all? If it's easier that's fine, but it also seems like it's okay to assume that if someone needs one they can add it later...maybe?

Reply:
You're right - I need to clean this up and move the app styles to the apps, but it isn't done yet, so I'm leaving this here as a reminder...